packages/orbit-components/src/utils/common/index.ts

```diff
@@ -1,15 +1,25 @@
-import type { FlattenSimpleInterpolation } from "styled-components";
+import type {
+  FlattenInterpolation,
+  FlattenSimpleInterpolation,
+  ThemeProps,
+} from "styled-components";
 import { css } from "styled-components";
 
 import type { ObjectProperty } from "../../common/types";
+import defaultTheme from "../../defaultTheme";
 
 export const firstToUpper = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
-export const defaultFocus = (): FlattenSimpleInterpolation => css`
-  outline: 2px solid Highlight;
-  outline: 2px solid -webkit-focus-ring-color;
+export const defaultFocus = (): FlattenInterpolation<ThemeProps<typeof defaultTheme>> => css`
+  ${({ theme }) => css`
+    outline: 2px solid ${theme.orbit.paletteBlueNormal};
```

Comment:
This changes all the focus, which removes the native color entirely. I think our mistake is using this for the selected state (as we do with Segmented switch). I don't think we should change this, the default focus is to keep. What we want to change is the color of the selected state. Therefore, the selected state can't use this. It must use its own outline that explicitly states blue

Reply:
It will lead us to do it in the future in the same way. I can expect, that we will receive the same request about InputField and other components. I do not see a problem to force it to be a blue color, do you? What's your argument for keeping the default color? The main reason why we did that before was, that box-shadow had not sufficient contrast as blue outline, but I personally do not mind to implicitly have that as blue

Reply:
I would keep the default focus color precisely for accessibility purposes. If it's something that can differ on the browser side for whatever reason (settings, default, etc.), I think we should keep it.
I'm all in favor of switching the selected/active state. But back in time designers agreed on going with the native colors, and I don't think that's meant to be changed. And if native colors can be so different… that's a consequence we should not worry about.
So the SegmentedSwitch option that is selected has our blue outline. But its focus color should be the native from whatever browser is being used. That was my understanding from the thread and that's what I think we should do, honestly

Reply:
Ok, how it's different from the native browser/OS/user's preferences colors? For A11y is important only contrast here, so I do not think it would lead to a11y issues. 

I prefer to add those styles based on the checked/unchecked state, it's closer to the default input behavior. As I mentioned above, it would lead us to change that probably in the future for every component, so I do not think it's good. 

I would probably change my opinion if you will convince me with some real arguments :) 

Designers agreed that before, because for them it was the same, I am not even sure that they got what we did, because reading the thread in slack, I can say, that Milan is not aware of how the outline works. If we told them before, that somewhere it will be black - I think they could change their opinion.

Reply:
My understanding is that that `-webkit-focus-ring-color` can change based on accessibility preferences configured by the user, that's why I raised accessibility questions. I am not certain it happens, but it would make sense, otherwise why would browsers define it differently amongst them? 💭 

The black outline was clearly undesired, but I understood they wanted native colors for focus, but not for active/selected, so that's what I was expected to see applied here.

Why can't we have a color for focus (the native one) and another for active/selected? 🤔

Reply:
Plus, it's weird to me, to have InputField with a black outline and SegmentedSwitch with blue. We either should do the same for all other Inputs, as you suggested in SegmentedSwitch or do it as I suggested and just update it inside defaultFocus.

Reply:
Maybe I wasn't clear, but in case we do it differently (focus vs active), we should do it for all, yes.
The black outline would occur only on browsers with it defined, but that's something we can't (and probably shouldn't) control. The active state is "ours" so we can put it blue as we want. Does it make sense?

But let's confirm with designers if focus is always equal to selected (and therefore different from the native) or if we should have native for focus and custom for selected 🙏

Reply:
I do not think that there are our/not our. Even the textarea you are currently writing it here is overwritten :D 
So I still prefer to implicitly set the color for the outline

Reply:
That's what GitHub's design system owners decide for themselves 😄 I am only raising the question because the change is quite recent. But it can be as you said: unintentional